src/views/Start/Start.vue

```diff
@@ -18,17 +22,44 @@ import log from 'loglevel'
 
 import BoxLoader from '../../components/helpers/BoxLoader'
 import { getOpenLoginInstance } from '../../openlogin'
+import { getIFrameOriginObject } from '../../utils/utils'
 
 export default {
   name: 'Start',
   components: { BoxLoader },
+  data() {
+    return {
+      whiteLabel: undefined,
+      isCustomVerifier: false,
+      iframeOrigin: {
+        href: '',
+        hostname: '',
+      },
+    }
+  },
+  computed: {
+    dappName() {
+      return this.isCustomVerifier ? this.whiteLabel?.name || this.iframeOrigin.hostname : 'Web3Auth'
+    },
+    dappUrl() {
+      return this.isCustomVerifier ? this.whiteLabel?.url || this.iframeOrigin.href : 'https://app.tor.us'
+    },
+    showConstructing() {
+      return (this.whiteLabel?.isActive && this.isCustomVerifier && this.dappName && this.dappUrl) || !this.whiteLabel.isActive || !this.whiteLabel
+    },
+  },
   async created() {
     try {
       const { loginProvider, state, skipTKey, ...rest } = this.$route.query
       const stateParams = JSON.parse(safeatob(state))
       log.info('logging in with', loginProvider, state, skipTKey, rest)
       const { whiteLabel, loginConfig = {} } = stateParams
+      this.iframeOrigin = getIFrameOriginObject()
```

Comment:
cannot get this here.
send it from bottom while opening start popup

Reply:
done